app-misc/mdate/mdate-1.7.0.3.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A converter to Mayan dates"
+DESCRIPTION="Mdate is a utility that converts Gregorian dates to Mayan Long Count dates. \
+It was originally an implementation of some algorithms published by N. Dershowitz and \
+E.M. Reingold in their book Calendrical Calculations."
+HOMEPAGE="https://github.com/ewe2/mdate"
+COPYRIGHT="2010 - 2014 Sean Dwyer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ewe2/mdate/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="6d58a8cf63c25b2b4b5b8904ba7c982f18608e8fa65436c555280699a020bcf8"
+PATCHES="mdate-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	mdate = $portVersion
+	cmd:mdate = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
+	cmd:ld
```

Comment:
We prefer to sort alphabetically the lines inside BUILD_PREREQUIRES.
Could you please move the "cmd:make" 2 lines below, just after the "cmd:ld" ?